src/mesh/coordinates.cxx

```diff
@@ -1027,21 +1106,9 @@ int Coordinates::jacobian() {
   // deriving from extrapolated covariant metric components
   J = interpolateAndExtrapolate(J, location, extrapolate_x, extrapolate_y);
 
-  // Check jacobian
-  bout::checkFinite(J, "The Jacobian", "RGN_NOCORNERS");
-  bout::checkPositive(J, "The Jacobian", "RGN_NOCORNERS");
-  if (min(abs(J)) < 1.0e-10) {
-    throw BoutException("\tERROR: Jacobian becomes very small\n");
-  }
```

Comment:
Did you mean to also remove this check?

Reply:
I did - I moved it along with the others into the `Coordinates` constructors on the basis that it might fail for the re-calculated Jacobian but pass for the read-in one, so if we have read one in, there is no error.

Reply:
My bad, I missed the new location!